tests/test_validator_integration.py

```
@@ -507,15 +543,17 @@ class TestRemoveUnusedColumns:
 
     def test_default_remove_unused_columns_is_true(self):
         """The HF default for remove_unused_columns is True."""
+        pytest.importorskip("torch", reason="requires torch")
         from gliner.training.trainer import TrainingArguments
 
         args = TrainingArguments(output_dir="/tmp/_test_ruc")
```

Ruff (0.15.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/_test_ruc"

(S108)